source/includes/json/_bulk_data_columns.md

```diff
@@ -13,12 +13,14 @@ which table header row is returned.
 id,petition_id,email,first_name,last_name,phone_number,postcode,created_at,join_organisation,deleted_at,unsubscribe_at,external_constituent_id,member_id,additional_fields,cached_organisation_slug,source,external_id,new_member,external_action_id,locale,bucket,country,updated_at,user_ip,confirmation_token,confirmed_at,confirmation_sent_at,last_signed_at,join_list_suppressed,old_daisy_chain_used,from_embed,user_agent,confirmed_reason,synced_to_crm_at,daisy_chain_experiment_slug,eu_data_processing_consent,from_one_click,consent_content_version_id,daisy_chain_id_used,email_opt_in_type_id,facebook_id,utm_params,postcode_id,referring_share_click_id,join_partnership,opt_in_sms,sms_opt_in_type_id,recaptcha_score,new_mobile_subscriber
 ```
 
-### HTTP Request
+#### HTTP Request
 
 `GET https://demo.controlshiftlabs.com/api/bulk_data/schema/columns?table=signatures`
 
-### Query Parameters
+#### Query Parameters
 
 Parameter |  Description
 --------- | ------------------
 table     |  Which table from the bulk data schema endpoint should be returned
+
+<div></div>
```

Comment:
Are these empty `div`s needed?

Reply:
Good question! These are a hack we already use elsewhere, to force the next section to start _after_ the JSON response stuff in the right column from this section. Without this, long JSON responses can stack up and not be aligned with the rest of the documentation for their endpoints.
I was just going to offer to add comments, but then I remembered that Markdown doesn't have proper comment support :disappointed: